docs/src/techpapers/netcdf_io.rst

```diff
@@ -0,0 +1,116 @@
+.. _netcdf_io:
+
+.. testsetup:: chunk_control
+
+    import iris
+    from iris.fileformats.netcdf.loader import CHUNK_CONTROL
+
+    from pathlib import Path
+    import shutil
+    import tempfile
+
+    tmp_dir = Path(tempfile.mkdtemp())
+    tmp_filepath = tmp_dir / "tmp.nc"
+
+    cube = iris.load(iris.sample_data_path("E1_north_america.nc"))[0]
+    iris.save(cube, tmp_filepath, chunksizes=(120, 37, 49))
+
+
+.. testcleanup:: chunk_control
+
+    shutil.rmtree(tmp_dir)
+
+
+=============================
+NetCDF I/O Handling in Iris
+=============================
+
+This document provides a basic account of how NetCDF files work within Iris.
+
+This document is still a work in progress, so might include blank or unfinished sections,
+watch this space!
+
+
+Chunk Control
+--------------
+
+Default Chunking
+^^^^^^^^^^^^^^^^
+
+Chunks are, by default, optimised by iris on load. This will automatically
+decide the best chunksize for your data without any user input.
```

Comment:
Would be good to mention what drives the decision:

https://github.com/SciTools/iris/blob/02feb2ac3310d9a0ecbd99f61c1b668414b8239d/lib/iris/_lazy_data.py#L69-L71

https://github.com/SciTools/iris/blob/02feb2ac3310d9a0ecbd99f61c1b668414b8239d/lib/iris/fileformats/netcdf/loader.py#L740-L746

Doesn't need to be word-for-word, but would be good to summarise some of it. I know this isn't behaviour that we have added, but this page is the first ever opportunity to document it in paragraph form!